app/controllers/notes_controller.rb

```diff
@@ -470,11 +470,13 @@ def new_note
   end
 
   def new_preview_note
-    Node.new_preview_note(uid: current_user.uid,
+    Node.new_preview_note(
++      uid: current_user.uid,
```

Comment:
```suggestion
      uid: current_user.uid,
```

Reply:
Same with this.

Reply:
Hello, am I understanding that the problem here is that I have to delete one space and bring it forward?

Reply:
@IdayatSanni The issue is that you added the plus sign in the code.